Automattic/jetpack
# JS Components: Add Mastodon icon

## Proposed changes:

* This PR adds a new Mastodon icon.

### Other information:


## Jetpack product discussion


* pewaj1-3x-p2.

## Does this pull request change what data or activity we track or use?


* No.

## Testing instructions:





* The icon is not being used yet, but there's some instructions on how to test this on pewaj1-3x-p2. The icon should also appear on the sharing sidebar:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/24195829/219145864-dd1cc627-0983-4c36-bacf-8c7423530c53.png">

* It is possible to check the new icon by checking out this branch, then running `cd projects/js-packages/storybook && pnpm run storybook:dev`.

* Initially, as can be seen on the image, the icon will not have the Mastodon color (#6364ff) but it was added on https://github.com/Automattic/wp-calypso/pull/72876. We'll need to update the package's version and dependency version on Jetpack as well.
